### ISSUE TITLE
Mines: copper node tracked as copper stone

### ISSUE THREAD

**Issue by danielw97** [stardew-access/stardew-access#273]
In the mines on 1.6, copper nodes are now identified as copper stones.
They also appear in the debris category of the object tracker whereas before I believe there was a dedicated mining category.
Tested on latest dev branch, commit b4d6262

**Comment by ParadoxiKat** [comment]
I can't reproduce this; are you using other mods? What specific floor?
I'm using latest dev build. I warped to floor 9 in the mines. I see "colord stones" in the debris category, which do sometimes drop copper, or stone, or other debris. Separately, I see "copper nodes" in the mine items category.
Is it possible you are mistaking "colored stones" for copper stones, or using a translation file that has mistranslated them as copper stones?
If not, please provide specific steps to reproduce this issue, and ideally a log file demonstrating it.
Thanks!

**Comment by danielw97** [comment]
Hi,
Thanks for the fast reply.
My screenreader is definitely saying "copper stone", I'm using English so I assume no translation is being done.
I can see it in the log like this:
[Stardew Access] Category: Debris | Object: Copper Stone
The mods I have installed are chests anywhere, cjb cheats, console commands, Kokoro, part of the community, project fluent, save backup, stardew access, and yet another fishing mod.
I'm happy to remove mods and test further, I've built from the dev branch and am running SDV 1.6.2
Happy to test a different version of the mod if that helps as well.
Also, no pressure at all on this issue, I'm sure there are a lot of other things that need fixing with 1.6.
Thanks.

**Comment by ParadoxiKat** [comment]
Hi,

I'm suspecting another mod is interfering. I'm developing using just stardew-access and it's dependencies, ProjectFluent and Kokoro. So with the two mase smapi mods, I have 5 mods loaded when I test / build at the moment.
I know my testers are using the fishing mod, and they absolutely would have reported this if it was happening for them. So I don't expect it's that one. Could you try removing mods other than those? So you should have 6 at the end:
The two base smapi mods, Yet another fishing mod, Kokoro and ProjectFluent, stardew-access.

**Comment by danielw97** [comment]
Hi again,
I've actually just done that a few moments ago, and am observing the same behavior with just Stardew Access and its dependencies.
I see that you're planning a new beta release shortly, shall I wait and test that to see if it's something with the way I'm building?
I've been building directly from the dev branch and everything else appears to be working fine.
I'm not sure if this might have anything to do with it, although I'm running SDV 1.6.2 which was released a few days ago.

**Comment by ParadoxiKat** [comment]
Turns out your build process and versions are all fine; this was a weird bug us devs and testers were unlikely to catch.
In fb33002a5e5ff1d2e38dbae00059b2f832e23fba, where we rebased the 1.6.0 alpha work over the 1.5.1 branch, the file `assets/TileData/ParentSheetIndexes.json` was accidentally renamed to `ParentSheedIndexes.json`. I have no explanation for why or how LOL.
For old players / testers / us developers, our old ParentSheetIndexes file with the correct name stayed around and was loaded, hence us not seeing this behavior. For players starting fresh with no old version to overwrite, this file was installed wrong and no old left over file was filling in the gap.
